src/hardware/vga_crtc.cpp

```diff
@@ -152,147 +192,173 @@ void vga_write_p3d5(io_port_t, io_val_t value, io_width_t)
 		VGA_StartResize();
 		break;
 
-	case 0x0A:	/* Cursor Start Register */
-		crtc(cursor_start)=val;
-		vga.draw.cursor.sline=val&0x1f;
-		if (IS_VGA_ARCH) vga.draw.cursor.enabled=!(val&0x20);
-		else vga.draw.cursor.enabled=true;
-		/*
-			0-4	First scanline of cursor within character.
-			5	Turns Cursor off if set
-		*/
+	case 0x0a: // Cursor Start Register
+		crtc(cursor_start)    = val;
+		vga.draw.cursor.sline = val & 0x1f;
+		if (IS_VGA_ARCH) {
+			vga.draw.cursor.enabled = !(val & 0x20);
+		} else {
+			vga.draw.cursor.enabled = true;
+		}
```

Comment:
This cleanup commit is so much nicer on the eyes with spaces and carriage returns. The packed-one-liners was just too dense to take in (for these old eyes, anyway!)

Reply:
Cheers. Yeah, just the logic is obscure and complex enough; there's no need to complicate it even further with bad style 😅  We should take all the small readability improvements we can get IMO.